src/pages/Settings/SettingsPage.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+	<www-page :gray-background="true">
+		<the-my-kiva-secondary-menu slot="secondary" />
+		<div class="title-area">
+			<div class="row column">
+				<h1>Settings</h1>
+			</div>
+		</div>
+		<div class="settings row column">
+			<section class="section">
+				<a
+					class="section__link"
+					href="/settings/account"
+				>
+					<h2 class="section__title">Account settings</h2>
+					<ul class="section__list">
+						<li>
+							<h3>Personal info</h3>
+							<p>
+								Your private account info: email,
+								<span v-if="!isMfaActive">password, </span>
+								mailing address
+							</p>
+						</li>
+						<li>
+							<h3>Lender profile</h3>
+							<p>What displays publicly on Kiva: name, location, occupation</p>
+						</li>
+					</ul>
+				</a>
+			</section>
+			<section
+				v-if="isMfaActive"
+				class="section"
+			>
+				<router-link
+					class="section__link"
+					to="/settings/security"
+				>
+					<h2 class="section__title">
+						Security settings
+					</h2>
+					<ul class="section__list">
+						<li>
+							<h3>Password</h3>
+							<p>Change your password</p>
+						</li>
+						<li>
+							<h3>2-step verification</h3>
+							<p>Turn on or off 2-step verification</p>
+						</li>
+					</ul>
+				</router-link>
+			</section>
+			<section class="section">
+				<a
+					class="section__link"
+					href="/settings/credit"
+				>
+					<h2 class="section__title">Credit settings</h2>
+					<ul class="section__list">
+						<li>
+							<h3>Repayment settings</h3>
+							<p>Where you want your money to go when it's repaid</p>
+						</li>
+						<li>
+							<h3>Inactivity settings</h3>
+							<p>Where you want your money to go if your account becomes inactive</p>
+						</li>
+						<li
+							v-if="isSubscriber"
```

Comment:
I don't think we need this conditional here for auto lend settings

Reply:
Well if we follow our other paradigms for handling AL when someone is subscribing to MG, this check should be the inverse so that the section doesn't show...

That said, leaving it off completely would allow someone to try and click into it which would then be subject to the redirect if they have and active MG.

Reply:
To match what we had in the monolith for this page, this should be `v-if="!isSubscriber"` so that it doesn't show when the user is signed up for monthly good.